app-arch/wimlib/wimlib-1.13.4.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools pax-utils
+
+DESCRIPTION="The open source Windows Imaging (WIM) library"
+HOMEPAGE="https://wimlib.net/"
+SRC_URI="https://wimlib.net/downloads/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="|| ( GPL-3+ LGPL-3+ ) CC0-1.0"
+SLOT="0"
+IUSE="cpu_flags_x86_ssse3 fuse iso ntfs openssl test threads yasm"
+REQUIRED_USE="cpu_flags_x86_ssse3? ( !openssl )"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	virtual/pkgconfig
+	cpu_flags_x86_ssse3? (
+		yasm? ( dev-lang/yasm )
+		!yasm? ( dev-lang/nasm )
+	)
+"
+RDEPEND="
+	dev-libs/libxml2:2
+	ntfs? ( sys-fs/ntfs3g )
+	fuse? ( sys-fs/fuse:0 )
+	iso? (
+		app-arch/cabextract
+		app-cdr/cdrtools
+	)
+	openssl? ( dev-libs/openssl:0= )
```

Comment:
There's no other issues here as I see, but sorting these would be nice too while at it :)